console/console-init/ui/src/modules/device/components/CreateDeviceButton/CreateDeviceButton.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React, { useState } from "react";
+import { Dropdown, DropdownToggle, DropdownItem } from "@patternfly/react-core";
+import { CaretDownIcon } from "@patternfly/react-icons";
+
+export interface ICreateDeviceButtonProps {
+  inputInfo: () => void;
```

Comment:
Just a suggession, would inputDeviceInfo and inputDeviceJSON make more sense here?

Reply:
Sounds fair